flocker/node/agents/blockdevice.py

```diff
@@ -1490,6 +1569,12 @@ def discover_state(self, node_state):
             NodeState(
                 uuid=self.node_uuid,
                 hostname=self.hostname,
+                # FLOC-1874 The sizes of the datasets for these manifestations
```

Comment:
This comment needs to be addressed, i.e. file a follow-up issue and link to it if you'd like. I don't think that this follow-up is necessary, personally - NodeState is reporting accurately.